### PR TITLE
Create component

### PR DESCRIPTION
**Pull request by Crash1212**

### CODE REVIEW

**Comment by Crash1212** [comment]
fixed tutorial

File: package.json [hunk 1]
```diff
@@ -12,17 +12,17 @@
...
     "react-dom": "^16.13.1",
     "react-redux": "^7.2.0",
     "react-scripts": "3.4.1",
+    "react-scripts-config": "^1.0.17-13",
     "react-scroll-up-button": "^1.6.4",
     "react-star-ratings": "^2.3.0",
     "redux": "^4.0.5",
-    "redux-thunk": "^2.3.0",
-    "react-scripts-config": "^1.0.17-13"
+    "redux-thunk": "^2.3.0"
```
Comment (Alan18081) [inline package.json:25]: Do we need `redux-thunk`? We use `redux-saga` instead

**Comment by Alan18081** [comment]
Please, resolve conflicts

File: package.json (hunk 1 above)
Comment (Crash1212) [reply package.json]: redux-thunk deleted

**Pull request merged**